wirecard/oxid-ee
# WDPD-64: Docker plugin volume

### This PR

* Updates the docker setup to mount the plugin folder and config file so that files can be edited without having to restart containers
* Updates `runtests_selenium` rake task to automatically reset the shop before testing

### How to Test

* Fire up the containers using `rake up`
* Run `runtests_selenium` and check if the tests are executed correctly

### Jira Links

* WDPD-64: https://jira.parkside.at/browse/WDPD-64
